src/pages/Home.js

```diff
@@ -1,10 +1,63 @@
 import React from 'react';
+import YourSvg from '../assets/hero.svg';
+import Category from '../components/Category';
+import { Link } from 'react-router-dom';
 
 function Home() {
   return (
-    <div>
-      <h2>This is the Home page</h2>
-    </div>
+    <>
+      <section className="hero">
+        <img class="hero-img" src={YourSvg} />
+
+        <h1 className="heading-3 center-text">
+          Best place to get
+          <span className="t-c-3 f-bold"> &nbsp;Brain</span> related stuff.
+        </h1>
+      </section>
+
+      <section className="categories">
+        <header className="categories-head z-index-lg">
+          <h3 className="heading-2 t-c-3 f-bold p-h-4">Categories</h3>
+          <Link to="/listing" className="heading-3 t-c-2 f-bold p-h-4 see-all">
+            See All
+            <span>
+              <i className="fa fa-arrow-right p-h-3"></i>
+            </span>
+          </Link>
+        </header>
+        <main className="category-container">
+          <Category
```

Comment:
Instead of writing <Category/> 5 times, you can make an array of objects and then loop here to put all details in 1 go.

Reply:
yes , will do at the last refactoring.